_blog.yml

```
@@ -1188,3 +1188,12 @@
   date: August 24, 2022
   tags:
     - research
+
+- local: megatron-training
+  title: "Training a Language Model with Megatron-LM"
```

I think "How to" blog posts worked quite well - maybe it's also just biased because those we done really well :) 

```suggestion
  title: "How to Train a Language Model with Megatron-LM"
```